qml/YubiKey.qml

```diff
@@ -208,6 +208,18 @@ Python {
         return false
     }
 
+    function hasAnyNonTouchTotpCredential() {
+        if (entries != null) {
+            for (var i = 0; i < entries.length; i++) {
+                if (!entries[i].credential.touch
+                        && entries[i].credential.oath_type === 'TOTP') {
+                    return true
+                }
+            }
+        }
+        return false
+    }
+
```

Comment:
I added this search logic in `qml/Util.qml:find(items, predicate)`, might want to reuse that. Probably also merge `Util.qml` into `utils.js` (which I didn't notice at the time), but I can do that in a separate PR.

Reply:
Cool, i'll look into it.